javascript/javascript1/week2/jsweek2.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lord " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2 = getFullname("Benjamin", "Hughes", false);
+
+console.log(fullname1);
+console.log(fullname2);
```

Comment:
Clean and concise implementation, good work 👌

If you want to experiment further, you might find it interesting to look into template literals for your return statements. 

How could you change your implementation to include other genders? 💡